pkgs/by-name/sp/spice-autorandr/package.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, autoconf
+, automake
+, libX11
+, libXrandr
+}:
+
+stdenv.mkDerivation rec {
+  name = "spice-autorandr";
```

name shouldn't be set directly

```suggestion
  pname = "spice-autorandr";
```